frontend/views/containers/group-settings/GroupVotingSystem.vue

```diff
@@ -0,0 +1,151 @@
+<template lang='pug'>
+  ul.c-wrapper
+    li.cardBox(
+      v-for='type in votingSytemSorted'
+      :class='{ isActive: isVotingActive(type) }'
+      :aria-current='isVotingActive(type)'
+    )
+      p.is-title-4 {{ config[type].title }}
+      p.has-text-1.c-expl(v-html='config[type].explanation')
+      i18n.pill.is-primary.c-active(v-if='isVotingActive(type)') Active
+
+      dl.c-status(v-if='isVotingActive(type)')
+        dt.c-status-term {{ config[type].status }}
+        dd.c-status-desc
+          span(v-html='votingValue(type)')
+          i18n.link(
+            tag='button'
+            @click='openVotingProposal(type)'
+          ) Propose change
+
+      banner-simple.c-banner(
+        severity='info'
+        v-if='isVotingRuleAdjusted(type)'
+      )
+        | * &nbsp;
+        i18n This value was automatically adjusted because your group is currently smaller than the disagreement number.
+
+      i18n.link(
+        v-if='!isVotingActive(type)'
+        tag='button'
+        @click='openVotingProposal(type)'
+      ) Propose changing to this voting system
+</template>
+
+<script>
+import sbp from '~/shared/sbp.js'
+import { RULE_PERCENTAGE, RULE_DISAGREEMENT } from '@model/contracts/voting/rules.js'
+import { OPEN_MODAL } from '@utils/events.js'
+import L from '@view-utils/translations.js'
+import BannerSimple from '@components/banners/BannerSimple.vue'
+
+export default {
+  name: 'GroupVotingSystem',
+  components: {
+    BannerSimple
+  },
+  data: () => ({
+    config: {
+      [RULE_PERCENTAGE]: {
+        title: L('Percentage based'),
+        explanation: L('Proposals are accepted when the required percentage of members agree to the proposal.'),
+        status: L('Percentage of members that need to agree:')
+      },
+      [RULE_DISAGREEMENT]: {
+        title: L('Disagreement number'),
+        explanation: L('Proposals are rejected when a certain number of members disagree with the proposal.'),
+        status: L('Maximum number of “no” votes:')
+      }
+    }
+  }),
+  computed: {
+    votingSytemSorted () {
+      return [RULE_DISAGREEMENT, RULE_PERCENTAGE] // TODO sort order based on groupSettings.
+    }
+  },
+  methods: {
+    openVotingProposal (type) {
+      sbp('okTurtles.events/emit', OPEN_MODAL, 'ChangeVotingProposal', { type })
+    },
+    isVotingActive (option) {
+      return option === this.votingSytemSorted[0]
```

Comment:
This should be based on the value of `store.getters.groupSettings.proposals[PROPOSAL_GROUP_SETTING_CHANGE].rule`.

Note: technically it can be based on the value of `store.getters.groupSettings.proposals[PROPOSAL_INVITE_MEMBER].rule` or `store.getters.groupSettings.proposals[PROPOSAL_REMOVE_MEMBER].rule` or any of the other proposal types, since we've decided that we are making it the same voting rule for all proposal types.

Since that's an easy change you can do it for this PR, or if you prefer you can add this as a TODO comment here.

Reply:
This PR does not contain any logic, even that simple thing. It will be done on a next PR :)

Reply:
OK but just to note forget can you add a TODO comment to this effect? You can copy/paste what I wrote above if you want.

Reply:
Sure... done